frontend/javascript/snippets/bootstrap.ts

```diff
@@ -1,6 +1,17 @@
-import 'bootstrap/js/dist/alert'
+// FIXME: init consistently across components, some don't work yet
 import 'bootstrap/js/dist/collapse'
```

Comment:
It seems like initializing Bootstrap components is different for every one now. Tooltips for instance don't work when just globally imported. Also, the non-module files contain unnecessary code for jQuery users. Haven't found an elegant way yet to initialize them nicely.

Reply:
I don't think that is necessary! I had a problem there as well but it was because two bootstrap's were running on the page, the second undoing the actions of the first. You don't need to manually setup bootstrap components, it should work with the data-attribute APIs.

Reply:
Tooltips don't work at all for me when imported globally 🤔

Reply:
Another gotcha: I don't know why but the `froide` repo has the tendency to stick to its own old bootstrap 4.6. In `fragdenstaat_de` check version in `node_modules/froide/node_modules/bootstrap/package.json`

Reply:
nope, it's the correct version :/